src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/CcdCaseUpdater.java

```diff
@@ -295,8 +295,7 @@ private String getErrorMessage(String service, String existingCaseId, String exc
                 exception.contentUTF8()
             );
             log.error(msg, exception);
-
-            return Optional.of(msg);
+            throw new CcdCallException(msg, exception);
```

Comment:
this is the last one with branch **not** returning a value so single return is from success path which is `optional.empty`. we can get rid of return type and make calling method simpler

Reply:
The same story says that in case of conflicts errors need to be returned to the caseworker. This means that we need a return value, just not in this place.

Reply:
@lgonczar As mentioned in the description for the PR
It appears that FeignException.Conflict is already handled as defined in the task.
It is retrown in `updateCaseInCcd` method and in `updateCase` method it is caught and converted in eror response.